unstructured/ingest/connector/astra.py

```diff
@@ -123,3 +130,117 @@ def normalize_dict(self, element_dict: dict) -> dict:
             "content": element_dict.pop("text", None),
             "metadata": element_dict,
         }
+
+
+@dataclass
+class AstraIngestDoc(IngestDocCleanupMixin, BaseSingleIngestDoc):
+    connector_config: SimpleAstraConfig
+    metadata: t.Dict[str, str] = field(default_factory=dict)
+    registry_name: str = "astra"
+
+    @property
+    def filename(self):
+        return (
+            Path(self.processor_config.output_dir)
+            / f"{self.metadata['_id']}.txt"
+        ).resolve()
+ 
+    @property
+    def _output_filename(self):
+        return Path(self.processor_config.output_dir) / f"{self.filename}.json"
+    
+    def update_source_metadata(self, **kwargs):
+        if not self.metadata:
+            self.source_metadata = SourceMetadata(
+                exists=False,
+            )
+            return
+        self.source_metadata = SourceMetadata(
+            exists=True,
+        )
+    
+    @SourceConnectionError.wrap
+    @requires_dependencies(["astrapy"], extras="astra")
+    @BaseSingleIngestDoc.skip_if_file_exists
+    def get_file(self):
+        with open(self.filename, "w") as f:
+            f.write(self.metadata["content"])
```

Comment:
I didn't see a "content" key in the data that I was downloading from the collection we are using.

Also, can you have each field's value write to a separate line like we do in Mongodb? Or were you mimicing a different source connector.

For mongodb connector the download .txt file just looks like this:

```
1901
Kansas Saloon Smashers
American
Unknown
nan
unknown
https://en.wikipedia.org/wiki/Kansas_Saloon_Smashers
```